wsgi/odyssy/odyssy/settings.py

```diff
@@ -114,3 +114,17 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(WSGI_DIR, 'static')
+
+
+# This will allow cookies  to be sent only via HTTPS connections.
+SESSION_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = True
+
+
+# Terminates the session as soon as browser closes.
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
```

Comment:
There is nothing wrong in doing this. Imagine that you have signed in from a cyber cafe and accidentally, you closed the window without signing out. If this option is set True, then as soon as browser closes, session will expire. This method is also used by IRCTC.